src/components/login-form.js

```diff
@@ -1,14 +1,135 @@
-import React from "react";
+import React, { useState } from "react";
+import axios from "axios";
 import styled from "styled-components";
 
-export default function LoginForm() {
-  return <Wrapper>안녕 나는 로그인 폼!</Wrapper>;
+export default function LoginForm(props) {
+  //State에 로그인에 필요한 데이터 저장
+  const [userData, setUserData] = useState({
+    email: "",
+    password: "",
+  });
+  const { loginSuccess } = props;
+  // 변수 이름 쉽게하기 위해
+  const { email, password } = userData;
+
+  const checkBlank = () => {
+    // 둘 중 하나라도 안 채워져 있을시 알림
+    if (email === "" || password === "") {
+      alert("빈칸 채워주세요!!");
+      return false;
+    } else {
+      return true;
+    }
+  };
+
+  // 로그인 시도
+  const tryLogin = () => {
+    if (checkBlank() === false) {
+      console.log("실패");
+      return;
+    }
+    axios
+      .post(process.env.API_HOST + "/auth/signin/", {
+        email: email,
+        password: password,
+      })
```

Comment:
post요청의  data는 userData이기 때문에 굳이 새로운 객체를 생성해서 data로 넘길 필요가 없을 것 같아요!. 아래와 같이 data로 userData를 넘겨주는 게 어떨까요?

```suggestion
      .post(process.env.API_HOST + "/auth/signin/", uesrData)
```